substrate-node/pallets/pallet-smart-contract/src/lib.rs

```diff
@@ -437,6 +500,83 @@ pub mod pallet {
             Self::_create_name_contract(account_id, name)
         }
 
+        #[pallet::weight(10_000 + T::DbWeight::get().writes(1))]
+        pub fn create_capacity_reservation_contract(
```

Comment:
rename `capacity_reservation_contract_create` so they will appear in order from the polka UI